docs/application/native/guides/connectivity/nfc.md

```diff
@@ -98,9 +98,15 @@ The main features of the NFC API include:
 
   ![HCE routing](./media/nfc_hce_routing.png)
 
+  > **Note**
+  >
+  > - HCE operations begin with the launch of HCE application in Tizen Application Framework. With the execution of these operations, the data transfer to the application needs more time.
+  > - In case, you consider time efficiency, it is recommend not to create HCE applications and communicate with the Secure Elements provided in the device.
+  > - Alternatively, ensure that the HCE application is always in the launch state.
+
 ## Prerequisites
 
-To enable your application to use the NFC functionality:
+To enable use of NFC functionality in your application:
 
 1. To use the NFC API (in [mobile](../../api/mobile/latest/group__CAPI__NETWORK__NFC__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__NETWORK__NFC__MODULE.html) applications), the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
```

Comment:
@jh8801jung
Kindly incorporate the following: 

**Action item 1:** 
**Global Comment** 
Change all numbered lists in the document as shown in the example:  
1. -> 1) 
2.-> 2)
3.-> 3) 
4.-> 4) 
.
.
10.-> 10)

**Action item 2:** 
**Global Comment** 
Remove the following line just below the diagram after **Note**

Pay attention to the following:

**Action item 3:**
 
Replace 
1. To use the NFC API (in [mobile](../../api/mobile/latest/group__CAPI__NETWORK__NFC__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__NETWORK__NFC__MODULE.html) applications), the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:

with: 

1) To use the NFC API in ([mobile](../../api/mobile/latest/group__CAPI__NETWORK__NFC__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__NETWORK__NFC__MODULE.html) applications), the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:

Reply:
@mijin85cho  now the whole prerequisites looks like this 

# Prerequisites

You must enable NFC functionality in your application and ensure:

1) To use the NFC API in ([mobile](../../api/mobile/latest/group__CAPI__NETWORK__NFC__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__NETWORK__NFC__MODULE.html) applications), the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:

Reply:
Question to @nawab-samsung 

>Action item 1:
Global Comment
Change all numbered lists in the document as shown in the example:

What is the reason that you decided to break the style rule of the list?
https://github.com/Samsung/tizen-docs/blob/master/styleguide/style.md#lists


>Action item 3:

What is the reason of moving the location of "("?
The original sentence means
"To use the NFC API in mobile and in wearable applications, " 
The "(in mobile and wearable applications)" is used as both mobile/wearable profiles are supporting NFC. We have 2 profiles for native NFC, so 2 API links are needed. One to mobile native, and one to wearable native. 
To add 2 links to here, as you can see "(in mobile and wearable applications)" is used. If only one profile supported the NFC, the link to API reference would have linked to "To use the NFC API," and no additional parenthesis would have added.

Please refer [this page](https://github.com/Samsung/tizen-docs/blob/master/docs/application/native/guides/connectivity/mtp.md), the example of only one profile (mobile in this case) support case.
![image](https://user-images.githubusercontent.com/9958374/54247059-ced67a80-457a-11e9-901a-242cabfc6d56.png)

Other [example page](https://github.com/Samsung/tizen-docs/blob/master/docs/application/native/guides/messaging/messages.md), showing the case of mobile/wearable profiles are supported.
![image](https://user-images.githubusercontent.com/9958374/54247099-02b1a000-457b-11e9-8493-c499df8f5c3f.png)



Regarding the reason of parenthesis, your recommendation is changing
 "To use the NFC API, the application... " to "To use the NFC API in, the application...",
and I don't understand the reason of the change.